products/rhel9/kickstart/ssg-rhel9-cis_workstation_l2-ks.cfg

```diff
@@ -91,6 +91,9 @@ clearpart --linux --initlabel
 part /boot --fstype=xfs --size=512
 part pv.01 --grow --size=1
 
+# Ensure /dev/shm is a separate partition
+part /dev/shm --fstype=tmpfs --fsoptions="nodev,nosuid,noexec" --size=2048
```

Comment:
We want to fit to 20GB for VM (our tests allocate 20GB for VM). Those extra 2GB for `/dev/shm` are over the limit and tests are not ready for that. Please, if possible, reduce other partition (e.g. `/`) and use that space there.

Also I don't think 2GB are needed, 512MB might be enough.

Reply:
I have adjusted those you requested.